### DISCUSSION TITLE
SVGs misbehave

### DISCUSSION THREAD

**Discussion by pikil** [quasarframework/quasar#13625] [General - Components / Directives / etc] [answered]
Hi Quasar team,
just a quick question if anyone faced the same issue. When the quasar-based project loads, it shows SVGs in a very odd way. More oddly is that once it's loaded, the browser starts showing it's icons in a weird way as well...
(see the video example)

Has anyone experienced the same behaviour?

https://user-images.githubusercontent.com/26779040/172248346-f02858e4-4947-4d82-92a3-b1badb16af10.mov

**Comment by MilosPaunovic** [comment]
Quasar can't affect browser icons, so it's probably `Chrome` related. Try reinstalling browser, latest version, see if it helps.

**Comment by metalsadman** [comment, marked as answer]
tested on chrome, firefox, safari. your site's loading fine. check if you have some rogue extension creating the issue.

**Comment by pikil** [comment]
Hmm, yeah, that must be one of the extensions... I'll give it a go :)
Thanks a lot!!!
